infrastructure/modules/lambda/output.tf

```diff
@@ -0,0 +1 @@
+#
```

Comment:
Do we need an empty file ? Or should we add it when we actually use it ?

Reply:
There is no use for it currently was thinking for future use but can remove and add later when needed.